tests/unit/helpers/questionnaires.test.js

```diff
@@ -208,24 +226,20 @@ describe('getUserQuestionnaires', () => {
     );
   });
 
-  it('should return an empty array if questionnaire is already answered', async () => {
+  it('should return expectations questionnaire', async () => {
```

Comment:
tu peux detailler le pourquoi -> `should return expectations questionnaire if course not started and questionnaire not answered`

Reply:
done